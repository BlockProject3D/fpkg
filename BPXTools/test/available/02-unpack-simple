export NAME="Unpack (SIMPLE)"
export COMMAND="-f test/available/test.bpx unpack"
export DESC="Test the unpack command"
export ADDITIONAL="diff LICENSE.txt ../LICENSE.txt"
export STATUS=0
export POST_COMMAND="rm ./LICENSE.txt"