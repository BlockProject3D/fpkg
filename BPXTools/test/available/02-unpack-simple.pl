$Test = {
    Name => "Unpack (SIMPLE)",
    Command => "-f test/available/test.bpx unpack",
    Description => "Test the unpack command",
    Status => 0
};

sub TestBegin {

}

sub TestEnd {
    my $res = EnsureEqual("LICENSE.txt", "../LICENSE.txt");
    unlink("LICENSE.txt");
    return $res;
}
