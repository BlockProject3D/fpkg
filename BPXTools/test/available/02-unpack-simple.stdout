Reading LICENSE.txt with 1517 byte(s)...
